### PR TITLE
Fix various PHPCS issues

### PR DESCRIPTION
**Pull request by paulschreiber**

### CODE REVIEW

**Comment by westonruter** [comment]
@paulschreiber if you re-base you recent pull requests against `develop` the build errors related to PHP 5.3 should go away.

Also, feel free to combine your commits into a single PR that fixes multiple phpcs issues in one go. No need for separate PRs for each commit.

**Comment by paulschreiber** [comment]
* I created separate PRs to make reviewing the diffs easier. I also wasn't sure which ones you would accept.
* This was branched off `develop`. What sort of rebasing is needed?

**Comment by westonruter** [comment]
> I created separate PRs to make reviewing the diffs easier. I also wasn't sure which ones you would accept.

Normally that's good, but these PRs are almost all PHPCS fixes which can be easily reviewed together. Having a separate PR for each just creates too much noise.

> This was branched off develop. What sort of rebasing is needed?

A fix just landed in `develop` which fixes the PHP 5.3 issue: https://github.com/Automattic/amp-wp/commit/a04782af5ac4870eb338a8f2eee193f15c6a3a06

So just `git pull --rebase https://github.com/Automattic/amp-wp.git develop; git push -f` and that should fix the build.

**Comment by westonruter** [comment]
Please then feel free to `git cherry-pick` each commit from your other PHPCS-related pull requests to amend to this PR, and then close the other PRs.

**Comment by paulschreiber** [comment]
I've rebased and cherry-picked the commits (except for #1000, as it's not clear what do there).

**Comment by paulschreiber** [comment]
@westonruter What's causing Travis CI to fail here?

**Comment by westonruter** [comment]
PHPCS errors: https://travis-ci.org/Automattic/amp-wp/jobs/351608818#L315

**Comment by paulschreiber** [comment]
This commit (be6a78a) doesn't touch `class-amp-base-embed-handler.php`, which generates the error.

**Comment by westonruter** [comment]
@paulschreiber there are multiple errors: https://travis-ci.org/Automattic/amp-wp/jobs/351608818#L315-L337

PHPCS via wp-dev-lib is looking at changes made to the entire diff in the PR, not individual commits in the PR. Because you added `public` to a method that lacks the necessary phpdoc, now that PHPCS error is being surfaced because it is related to a line you touched.

**Comment by paulschreiber** [comment]
Odd. We saw a green checkmark after the previous commit (df77083). Do you want me to revert 78e64f4 or add docs?

**Comment by westonruter** [comment]
Adding the doc comments would be great.

**Comment by paulschreiber** [comment]
I've added many doc comments. PHPCS is still finding a few errors:
```
class-amp-base-embed-handler.php:59:32: error - Object property "DEFAULT_WIDTH" is not in valid snake_case format
class-amp-youtube-embed.php:161:13: warning - This comment is 50% valid code; is this commented out code?
class-amp-youtube-embed.php:170:13: warning - This comment is 47% valid code; is this commented out code?
test-amp-image-dimension-extractor.php:66:1: warning - Best practice suggestion: Declare only one class in a file.
test-amp-image-dimension-extractor.php:130:1: warning - Best practice suggestion: Declare only one class in a file.
```

Should we leave those for now?

File: includes/class-amp-theme-support.php [hunk 1]
```diff
@@ -229,7 +229,7 @@ public static function add_hooks() {
 		 * Disable admin bar because admin-bar.css (28K) and Dashicons (48K) alone
 		 * combine to surpass the 50K limit imposed for the amp-custom style.
 		 */
-		add_filter( 'show_admin_bar', '__return_false', 100 );
+		add_filter( 'show_admin_bar', '__return_false', 100 ); // phpcs:ignore
```
Comment (westonruter) [inline includes/class-amp-theme-support.php]: This actually isn't necessary. The PHPCS ruleset does not include the [AdminBarRemoval](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/VIP/AdminBarRemovalSniff.php) WordPress.com VIP sniff. You should make sure to use the project's own [ruleset](https://github.com/Automattic/amp-wp/blob/develop/phpcs.xml) when developing.

File: includes/embeds/class-amp-youtube-embed.php [hunk 2]
```diff
@@ -55,23 +83,49 @@ public function shortcode( $attr ) {
...
 
-		return $this->render( array(
-			'url' => $url,
-			'video_id' => $video_id,
-		) );
+		return $this->render(
+			array(
+				'url'      => $url,
+				'video_id' => $video_id,
+			)
```
Comment (westonruter) [inline includes/embeds/class-amp-youtube-embed.php]: Aside: things like this are more elegant as `compact( 'url', 'video_id' )`

Comment (paulschreiber) [reply includes/embeds/class-amp-youtube-embed.php]: Agreed that compact is better. `phpcbf` reformatted it as above (expanded).

**Pull request merged**